app/src/main/java/xyz/jhughes/laundry/activities/LocationActivity.java

```diff
@@ -37,18 +40,27 @@
     @Bind(R.id.location_activity_toolbar) Toolbar toolbar;
     @Bind(R.id.progressBar) ProgressBar mLoadingProgressBar;
 
-
     private LocationAdapter adapter;
 
     @Override
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
+
```

Comment:
I thought there was a difference between onNewIntent and onCreate() being called. OnCreate being on the first open and onNewIntent or something being called when a currently opened activity gets a new call. 

So we would only want it in onCreate() I guess. Maybe what you have is correct, hmm.